frontend/libs/studio-components/src/components/StudioPageHeader/StudioPageHeaderButton/StudioPageHeaderButton.module.css

```diff
@@ -0,0 +1,32 @@
+.button {
+  color: var(--fds-semantic-background-default);
+}
+
+.regularDark {
+  background-color: var(--studio-button-regular-dark);
```

Comment:
I may have been wrong, but does `--studio-button-regular-dark` token work here because `StuduioPageHeader` it's the children of a parent who has the className `studioPageHeader`?

Reply:
Correct 😄 
As the variable is declared at the "root" file in the StudioPageHeader, it's accessible in the children components.